validators/terravaloper16f9xsgwvw93pzugeztz8njmlpcveh0cw2rg3jn/profile.json

```diff
@@ -0,0 +1,8 @@
+{
+  "contact": {
+    "email": "f3hd@gmail.com",
+    "telegram": "",
+    "twitter": "",
+    "website": ""
```

Comment:
@GDeppe You can go ahead and remove these lines if there are no values to add here

Reply:
This has now been completed and resubmitted